_howdoi/how-do-i-launch-a-jupyter-notebook.adoc

```diff
@@ -0,0 +1,65 @@
+= launch a Jupyter notebook on OpenShift
+
+There are multiple ways to launch a Jupyter notebook on OpenShift with the
+radanalytics.io tooling. You can use the OpenShift console or the `oc` command
+line tool. Before running these instructions you will need to have installed
+the radanalytics.io manifest as referenced on the
+link:/get-started[Get Started] page.
+
+== Launching Jupyter from the OpenShift console
+
+To begin, select the "Jupyter Notebook" template from the "Add to Project"
+menu in your OpenShift console.
+
+pass:[<img src="/assets/howdoi/jupyter-launch-1.png" alt="Jupyter template" class="img-responsive">]
+
+After selecting the template and clicking past the information screen, you
+will be presented with several options for your notebook deployment. You will
+need to create a password for your notebook and optionally give it a name and
+the URI of a notebook file to download.
+
+pass:[<img src="/assets/howdoi/jupyter-launch-2.png" alt="Jupyter options" class="img-responsive">]
+
+After clicking "create" you will see the application deploy in your OpenShift
+project, simply click on the route link created and you will be directed to
+the notebook.
+
+pass:[<img src="/assets/howdoi/jupyter-launch-3.png" alt="Jupyter created" class="img-responsive">]
+
+== Launching Jupyter from the command line
+
+There are two methods for launching a Jupyter notebook from the command line:
+using the template, and directly from the container image.
```

Comment:
nit: no comma.  maybe "by using the template or directly with `oc new-app` and the container image"